### ISSUE TITLE
DJI site doesn't like cleanlinks

### ISSUE THREAD

**Issue by jawz101** [Cimbali/CleanLinks#122]
example page:

https://www.dji.com/mavic-air-2

looks like their site uses some domain called `www.js` that makes rules difficult

**Comment by Cimbali** [comment]
Yes, I’ve seen a number of websites doing this. Having resources called `www.js` or `www.css`. Sometimes there are also valid-looking names, such as `www.js.89e8d24e.chunk.css`.

Not sure if we should handle those specifically (we could detect invalid domains such as `www.{tld}`) or add rules as we encounter them.
